test/lsp/utils/text_edit.vimspec

```diff
@@ -582,11 +604,52 @@ Describe lsp#utils#text_edit
                 \     },
                 \     "newText": "import java.util.ArrayList;\n"
                 \   }
-                \ ])
+                \ ],
+                \ {'show_edits': 0})
 
             let l:buffer_text = s:get_text()
             Assert Equals(l:buffer_text, ['package main', '', 'import java.util.ArrayList;', '', 'public class Main {}', ''])
         End
+
+        It populates location list with changes
+            call lsp#utils#text_edit#apply_text_edits(
+                \ expand('%'),
+                \ [
+                \ {
+                \     "range": {
+                \       "start": {
+                \         "character": 0,
+                \         "line": 3
+                \       },
+                \       "end": {
+                \         "character": 0,
+                \         "line": 3
+                \       }
+                \     },
+                \     "newText": "import java.util.LinkedList;"
+                \   },
+                \   {
+                \     "range": {
+                \       "start": {
+                \         "character": 0,
+                \         "line": 2
+                \       },
+                \       "end": {
+                \         "character": 0,
+                \         "line": 2
+                \       }
+                \     },
+                \     "newText": "import java.util.ArrayList;"
+                \   }
+                \ ],
+                \ {'show_edits': 1})
+
+            let l:loclist = getloclist(0)
+            Assert Equals(l:loclist, [
+                \ {'lnum': 3, 'bufnr': 21, 'col': 1, 'pattern': '', 'valid': 1, 'vcol': 0, 'nr': 0, 'type': '', 'module': '', 'text': 'import java.util.ArrayList;'},
```

Comment:
You shouldn't check the value of `bufnr` directly, because it changes from run to run. This is also why CI is failing

Reply:
Yeah makes sense, what about https://github.com/prabirshrestha/vim-lsp/pull/555/commits/1d888699ade5344c01d846a4f66fa48999a7b156